std/experimental/ndslice/package.d

```diff
@@ -57,6 +57,12 @@ Multidimensional view is presented by $(SUBREF slice, Slice) type.
 ------
 auto matrix = new double[12].sliced(3, 4);
```

Comment:
FYI, https://github.com/D-Programming-Language/dmd/pull/5465 has been merged, so you can now use ddoc'd unittests for the module docs.